_submit_split.sh

```diff
@@ -1,4 +1,4 @@
-#! /bin/bash
+#! usr/bin/env bash
```

Comment:
Hi, sorry for the random drive-by comment.
I just noticed this small typo:
```suggestion
#! /usr/bin/env bash
```